impeller/renderer/backend/vulkan/driver_info_vk.cc

```diff
@@ -197,8 +329,20 @@ bool DriverInfoVK::IsEmulator() const {
 }
 
 bool DriverInfoVK::IsKnownBadDriver() const {
-  if (vendor_ == VendorVK::kQualcomm && driver_name_ == kAdreno630) {
-    return true;
+  if (adreno_gpu_.has_value()) {
+    auto adreno = adreno_gpu_.value();
+    switch (adreno) {
+      // see:
+      // https://github.com/flutter/flutter/issues/154103
+      //
+      // Reports "VK_INCOMPLETE" when compiling certain entity shader with
+      // vkCreateGraphicsPipelines, which is not a valid return status.
+      // See https://github.com/flutter/flutter/issues/155185 .
```

Comment:
This version of the PR doesn't actually affect that issue, right?

That is, this adds parsing but it doesn't yet change what set of GPUs get treated as known-bad.

Reply:
correct

Reply:
we're still going to try to aquire a device with a similar GPU and see what isn't working first.